adam-python/bdgenomics/adam/test/alignmentRecordRdd_test.py

```diff
@@ -201,3 +202,23 @@ def test_filterByOverlappingRegions(self):
 
         filtered = reads.filterByOverlappingRegions(querys)
         self.assertEquals(filtered.toDF().count(), 2)
+
+    def test_caching(self):
+
+        readsPath = self.resourceFile("unsorted.sam")
+        ac = ADAMContext(self.ss)
+
+        reads = ac.loadAlignments(readsPath)
+
+        reads.cache()
+        reads.unpersist()
```

Comment:
What are these tests doing?

Reply:
These are just testing that they can run, I dont know a better way to detect whether something is cached